.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "3.6"
+cache: pip
+install:
+  - pip install -r requirements.txt
+script:
+  - python ./init.py

```

Comment:
@Mansi145 - Basically, the command you've used to lint the code will be here instead of `python ./init.py` - say, `flake8 *.py` 😄